src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
And about: https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L264

Well, why sprintf when you can simply do:
```
throw new ClientException($scheme .' is not a valid scheme');
```

Or:
```
throw new ClientException("{$scheme } is not a valid scheme");
```